server-api/pom.xml

```diff
@@ -39,6 +39,7 @@
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-common</artifactId>
+            <version>3.3.0</version>
```

Comment:
The version is defined in the parent pom.
And it is updated to 3.3.0 with 80b305f827092f5e006d556aba3e02acee052c2d

Reply:
This PR could be closed